### PR TITLE
Add support for Skip Expressions to HTTP Tasks #1909

### PR DESCRIPTION
**Pull request by unamanic**
Adds support for Skip Expressions to BPMN HTTP Tasks during to the flowable-http module and Modeler.

#### Check List:
* Unit tests: YES
* Documentation: NO (Skip Expression not documented in other tasks that use them)

### CODE REVIEW

File: modules/flowable-http/src/main/java/org/flowable/http/bpmn/impl/HttpActivityBehaviorImpl.java [hunk 1]
```diff
@@ -141,84 +143,95 @@ public boolean verify(String s, SSLSession sslSession) {
...
-            }
+        boolean isSkipExpressionEnabled = SkipExpressionUtil.isSkipExpressionEnabled(skipExpressionText,
+                execution.getCurrentActivityId(), execution, commandContext);
 
-        } catch (Exception e) {
-            if (e instanceof FlowableException) {
-                throw (FlowableException) e;
-            } else {
-                throw new FlowableException(HTTP_TASK_REQUEST_FIELD_INVALID + " in execution " + execution.getId(), e);
+        if (skipExpressionText == null || !isSkipExpressionEnabled || !SkipExpressionUtil.shouldSkipFlowElement(skipExpressionText,
```
Comment (tijsrademakers) [inline modules/flowable-http/src/main/java/org/flowable/http/bpmn/impl/HttpActivityBehaviorImpl.java]: skipExpressionText == null check is not necessary here, can you remove this?

File: modules/flowable-http/src/test/java/org/flowable/http/bpmn/HttpServiceTaskTest.java [hunk 2]
```diff
@@ -497,6 +500,31 @@ public void testDeleteResponseEmptyBody() {
         assertNull(runtimeService.getVariable(processInstanceId, "myResponse"));
     }
 
+    @Test
+    @Deployment
+    public void testGetWithVariableNameAndSkipExpression() {
+        Map<String, Object> variables = new HashMap<>();
+        variables.put("_ACTIVITI_SKIP_EXPRESSION_ENABLED", true);
```
Comment (tijsrademakers) [inline modules/flowable-http/src/test/java/org/flowable/http/bpmn/HttpServiceTaskTest.java]: can you change this to _FLOWABLE_SKIP_EXPRESSION_ENABLED, and the same for the other lines using this variable?

**Comment by tijsrademakers** [comment]
Thanks for the PR, I've added two small comments, could you have a look and update the PR?

File: modules/flowable-http/src/main/java/org/flowable/http/bpmn/impl/HttpActivityBehaviorImpl.java (hunk 1 above)
Comment (unamanic) [reply modules/flowable-http/src/main/java/org/flowable/http/bpmn/impl/HttpActivityBehaviorImpl.java]: Removed redundant null check and updated test to ensure that Http Tasks without a skip expression are still executed when _FLOWABLE_SKIP_EXPRESSION_ENABLED is set.

File: modules/flowable-http/src/test/java/org/flowable/http/bpmn/HttpServiceTaskTest.java (hunk 2 above)
Comment (unamanic) [reply modules/flowable-http/src/test/java/org/flowable/http/bpmn/HttpServiceTaskTest.java]: Updated

**Pull request merged**